collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -121,6 +121,11 @@ protected FECWrapper doInBackground(String... path) {
         errorMsg = null;
 
         final String formPath = path[0];
+        if (formPath == null) {
+            Timber.e("formPath is null");
```

Comment:
@grzesiek2010 Do you think it would be useful to set the value of `errorMsg` to `formPath is null` as well. Because if not provided, then the message would appear to be `Sorry, unable to parse form.`

As you said that it occurs rarely so I am assuming that this is a recoverable exception. If we keep the message different, maybe someone in the future would attempt to fix it rather than just leaving it as is.

Reply:
Sounds like a good plan to me. You could even consider something like "formPath is null, please post on the forum with a description of what you were doing when this happened." Note that errorMsg values are deliberately not translated since they aren't expected to show up often.

Reply:
Thanks, done!